### PR TITLE
Write documents about the database tables

### LINKED ISSUE #168: Add document for the database tables

### PR DESCRIPTION
**Pull request by Huy-Ngo**
Despite the branch name this is probably not the last time we have to edit this section.
This resolves #168

### CODE REVIEW

**Comment by McSinyx** [comment]
Why do you need to assign people to a PR?

**Comment by Huy-Ngo** [comment]
Practice I learned from GitLab where requesting reviews = assigning the MR to reviewers and after you request changes you assign back to the MR author.

File: docs/source/meth/database/tables/index.rst [hunk 1]
```diff
@@ -4,7 +4,40 @@ Data Tables
...
+For example, two entities ``Artifact``\s storing revisions for report and slides
+are included within a table row.
+``Task``\s are stored in an array in the project.
+.. TODO write about secondary index here and delete what Phong wrote
+
+.. figure:: images/project.svg
    :figwidth: 100%
 
-   The structure of data table ``projects``
+   The structure of database table ``projects``. [1]_
```
Comment (McSinyx) [inline docs/source/meth/database/tables/index.rst:31]:
```suggestion
   The structure of database table ``projects``. [#1]_
```

Comment (Huy-Ngo) [reply docs/source/meth/database/tables/index.rst:31]: Why tho?

**Review by McSinyx — APPROVED** [review]

File: docs/source/meth/database/tables/index.rst (hunk 1 above)
Comment (McSinyx) [reply docs/source/meth/database/tables/index.rst:31]: My mistake, was looking for it and thought it wasn't rendered.

**Comment by McSinyx** [comment]
Gotta be some org's convention, because normally contributors don't have the permission to assign.

**Comment by Huy-Ngo** [comment]
I assume no change requests mean approval

**Pull request merged**